index.html

```diff
@@ -0,0 +1,130 @@
+                <h4 class="card-title title-web-design">Web Design</h4>
+                <p class="program-desc">Focus on website building, Design, Publication and everything else  you need to launch your own website.</p>
+            </div>
+            <div class="card mobile-app">
+                <img src="images/program_icon_03.png" class="icon mobile-icon" alt="Mobile App" />
+                <h4 class="card-title title-mobile app">Mobile App</h4>
+                <p class="program-desc">Ladies will design, build and program their own mobile app using the knowledge they got from the discussions.</p>
+            </div>
+            <div class="card discussion">
+                <img src="images/program_icon_04.png" class="icon discussion-icon" alt="Discussion" />
+                <h4 class="card-title title-discussion">Discussion</h4>
+                <p class="program-desc">Discussions with feature leading technology experts and professionals from all over the country.</p>
+            </div>
+            <div class="card community">
+                <img src="images/program_icon_05.png" class="icon community-icon" alt="Community" />
+                <h4 class="card-title title-community">Community</h4>
+                <p class="program-desc">Introduction and Registration to join the communities and technology groups from across the country by experts.</p>
+            </div>
+            <button type="submit" class="button-1">Join PyLadies Namibia 2023</button>
+        </div>
+        <a class="link-1" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <!-- Featured Speakers Section-->
+    <section class="speakers">
+        <h2 class="speaker-title">Featured Speakers</h2>
+        <hr>
+        <div class="speakers-container"></div>
+
+    </section>
+
+    <!-- Partners Section -->
+    <section class="partners">
+        <h2 class="partner-title">Partner</h2>
+        <hr>
+        <div class="partners-list">
+            <a rel="noopener noreferrer" target="_blank" href="https://www.mozilla.org/en-US/"><img src="images/mozilla.svg" alt="Mozilla"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.google.com/en-US/"><img src="images/google.svg" alt="Google"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.naver.com/en-US/"><img src="images/naver.svg" alt="Naver"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.daumkakao.com/en-US/"><img src="images/daumkakao.svg" alt="Daumkakao"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.airbnb.com/en-US/"><img src="images/airbnb.svg" alt="Airbnb"></a>    
+        </div>
+    </section>
+
+    <!-- Footer Section -->
+    <section class="footer">
+        <a href="index.html"><img src="images/logo.jpg" alt="pyladies-logo"></a>
+        <p class="footer-desc">2022 PyLadies Namibia <br> Same right reserved</p>
+    </section>
```

Comment:
Kindly replace the `section` tag here with `footer` tag since this part is the footer of webpage 🤗